src/main.rs

```diff
@@ -14,17 +16,22 @@ struct Args {
     count: u8,
 }
 
-fn main() {
-    let _args = Args::parse();
+#[tokio::main]
+async fn main() {
+    let static_files = warp::fs::dir("static"); 
 
+    let routes = warp::path("app").and(static_files);
+
+    warp::serve(routes).run(([127, 0, 0, 1], 3030)).await;
```

Comment:
No need for hosting anything. We want just scaffold the corresponding application